packages/core/src/module.js

```diff
@@ -16,6 +16,11 @@ export default class ModuleContainer {
     // Call before hook
     await this.nuxt.callHook('modules:before', this, this.options.modules)
 
+    if (this.options._start) {
```

Comment:
I think this condition should be like `if (!...)`

Reply:
PS: Seems good idea to run devModules **before** modules :+1:

Reply:
Ops, updated the check -- will try and add a test next :)